### ISSUE TITLE
Latency 

### ISSUE THREAD

**Issue by Blueshawk** [atareao/Touchpad-Indicator#52]
Long latency between typing and activation makes it fairly useless and frustrating. Palming causes me to routinely type in random places --wherever the mouse cursor happens to be.
HP15-db0069wm
AMD Ryzen 5 2500
ATI vega9 gpu
Ubuntu 18.04LTS  (peppermint 9)

**Comment by sgon00** [comment]
The long latency does bother me too. `Milliseconds to wait after the last key press before enabling the touchpad` doesn't take any effects when I set it less than 2000. I tried 100, it still waits like 2s. That is too long.

**Comment by kpldvnpne** [comment]
Yes. Also, the latency between typing and touchpad activation would have been acceptable with normal typing. What makes it more frustrating for me is that, when you are using KeyBoard shortcuts like copy (Ctrl + C) and paste (Ctrl + V), your touchpad is disabled for a very long time, which is very unhelpful.

**Comment by mg979** [comment]
@sgon00 @kpldvnpne the issue is about latency between typing and touchpad deactivation, not the wait before reactivation. And I agree with @Blueshawk that this high latency makes this program useless for me too.
